js/iframe-coordinator/tsconfig.json

```diff
@@ -3,9 +3,10 @@
     "outDir": "./",
     "noImplicitAny": true,
     "module": "commonjs",
-    "target": "es5",
-    "allowJs": true
+    "declaration": true,
+    "declarationDir": "./",
+    "target": "es5"
   },
-  "include": ["**/*.ts"],
-  "exclude": ["node_modules"]
+  "include": ["libs"],
```

Comment:
This helps typescript to generate the files into the root dir.  If you use the glob pattern, it keeps the src directory.

The alternative would to have a copy step to move them to root.